dynamodb_utils/dumper.py

```diff
@@ -1,93 +1,81 @@
 #!/usr/bin/env python
 # coding=utf-8
-from __future__ import absolute_import, print_function
```

Comment:
there's no point changing this line unnecessarily in this commit, and at some point I'll add my full isort config which forces all the `__future__` imports in all files for maximum python 2->3 compat anyway